parse.go

```diff
@@ -23,6 +23,7 @@ type Issue struct {
 	IssueBackground       string            `xml:"issueBackground"`
 	RemediationBackground string            `xml:"remediationBackground"`
 	IssueDetail           string            `xml:"issueDetail"`
+        IssueDetailItems      []string          `xml:"issueDetailItems>issueDetailItem"`
```

Comment:
I know it's a little detail, but can you run go fmt on the code so everything is aligned.

Reply:
Normally would have, did this in-browser, all looked aligned, but seems not.